src/estimation/include/iDynTree/Estimation/BipedFootContactClassifier.h

```diff
@@ -116,6 +116,12 @@ namespace iDynTree
          * @param pattern switching pattern
          */
         void setContactSwitchingPattern(SwitchingPattern pattern) { m_pattern = pattern; } 
+        
+        /**
+         * set  primary foot
+         * @param foot primary foot
+         */
+        void setPrimaryFoot(contactFoot foot) { m_primaryFoot = foot; }
```

Comment:
What happens if I call `setPrimaryFoot(UNKNOWN_FOOT)`?

Reply:
This method was mainly intended to be called by an external process to set the primary foot in the initial setting, before any contact is broken. In case it is set to UNKNOWN_FOOT, it waits for the foot normal force measurements (checks left first and then right), and activates corresponding foot, handled in the detectTransitions() method. In my opinion, this could be handled in a better way. But, I am not able to think how. Do you have any suggestions ?

Reply:
The current behavior is ok for me, can you just document it in a method? Just copy&paste the phrase you used in this comment is ok, thanks.